frontend/public/components/service-account.jsx

```diff
@@ -15,37 +15,38 @@ const KubeConfigify = (kind, sa) => ({
   label: 'Download kubeconfig file',
   weight: 200,
   callback: () => {
-    const name = _.get(sa, 'secrets[0].name');
+    const secretName = _.get(sa, 'secrets[0].name');
```

Comment:
Existing bug:  this requires the token secret come first.  If not, an error occurs (e.g., openshift-monitoring/alertmanager-main)

<img width="607" alt="screen shot 2019-03-07 at 2 18 08 pm" src="https://user-images.githubusercontent.com/895728/53982819-dbbf1e00-40e3-11e9-88e3-e44220baa4e7.png">